### PR TITLE
Add job to run TestGrid UI tests.

### PR DESCRIPTION
**Pull request by michelle192837**
Copied from GCP/testgrid: https://github.com/GoogleCloudPlatform/testgrid/blob/main/.prow.yaml#L15

(WIP: I need to make some adjustments)

### CODE REVIEW

File: config/jobs/kubernetes-sigs/testgrid/testgrid-presubmits.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+presubmits:
+  kubernetes-sigs/testgrid:
+  - name: test-testgrid-npm
+    cluster: eks-prow-build-cluster
+    decorate: true
+    always_run: true
+    spec:
+      containers:
+      # See https://playwright.dev/docs/ci
+      - image: mcr.microsoft.com/playwright:v1.32.0-focal
```
Comment (ameukam) [inline config/jobs/kubernetes-sigs/testgrid/testgrid-presubmits.yaml:10]:
```suggestion
      - image: mcr.microsoft.com/playwright:v1.47.0
```

Comment (michelle192837) [reply config/jobs/kubernetes-sigs/testgrid/testgrid-presubmits.yaml:10]: I think we need it at the same version as the one in the repo, so this should be updated soon but I don't think we want to update it until  then.

Comment (ameukam) [reply config/jobs/kubernetes-sigs/testgrid/testgrid-presubmits.yaml:10]: I can take care of bump the deps. I kind of want to use a more modern packages manager like `pnpm`.

Comment (michelle192837) [reply config/jobs/kubernetes-sigs/testgrid/testgrid-presubmits.yaml:10]: That sounds good to me, as long as you're able to run the UI locally already 👍 (I did verify the existing k8s-sigs/testgrid/web npm targets should work already).

Comment (michelle192837) [reply config/jobs/kubernetes-sigs/testgrid/testgrid-presubmits.yaml:10]: I'll go ahead and commit the 1.37.0 for now, assuming that's what you're targeting. (It looks like there's 1.48.0 as the latest version of the image, I assume if we upgrade to that it'll be later?)

Comment (ameukam) [reply config/jobs/kubernetes-sigs/testgrid/testgrid-presubmits.yaml]: Yes. Starting with 1.38, playwright no longer automatically download browsers for testing. We will need to adjust the scripts before we bump again.

**Pull request merged**